autopush/endpoint.py

```diff
@@ -473,7 +476,9 @@ def put(self, api_ver="v0", token=None):
                           api_ver,
                           crypto_key_header)
         d.addCallback(self._process_auth)
-        d.addCallback(self._token_valid)
+        d.addCallback(self._token_valid,
+                      caller="%s.%s" % (self.__module__,
+                                        self.request.method.lower()))
```

Comment:
We don't. This method is used for both "put" and "post". Pulling the method at least gives us that.